Constitution.txt

```diff
@@ -1,490 +1,264 @@
 
-14: To make Rules for the Government and Regulation of the land and naval Forces;
-
-15: To provide for calling forth the Militia to execute the Laws of the Union, suppress Insurrections and repel Invasions;
-
-16: To provide for organizing, arming, and disciplining, the Militia, and for governing such Part of them as may be employed in the Service of the United States, reserving to the States respectively, the Appointment of the Officers, and the Authority of training the Militia according to the discipline prescribed by Congress;
-
-17: To exercise exclusive Legislation in all Cases whatsoever, over such District (not exceeding ten Miles square) as may, by Cession of particular States, and the Acceptance of Congress, become the Seat of the Government of the United States, and to exercise like Authority over all Places purchased by the Consent of the Legislature of the State in which the Same shall be, for the Erection of Forts, Magazines, Arsenals, dock-Yards, and other needful Buildings;—And
-
-18: To make all Laws which shall be necessary and proper for carrying into Execution the foregoing Powers, and all other Powers vested by this Constitution in the Government of the United States, or in any Department or Officer thereof.
+13: To make all Laws which shall be necessary and proper for carrying into Execution the foregoing Powers, and which shall be within the scope of the foregoing Powers. It shall be the duty of all Representatives and Senators, the President of these United States, all State Executives, and State Legislatures, to ensure before their approval that a Bill under consideration falls within the scope of the foregoing powers, and is not in violation of any term of this Constitution.
 
 Section 9
-1: The Migration or Importation of such Persons as any of the States now existing shall think proper to admit, shall not be prohibited by the Congress prior to the Year one thousand eight hundred and eight, but a Tax or duty may be imposed on such Importation, not exceeding ten dollars for each Person.
+1: The Privilege of the Writ of Habeas Corpus shall not be suspended, unless when in Cases of Rebellion or Invasion the public Safety may require it, whereupon it shall be suspended for more than one week only with the approval of three-fourths of both the House of Representatives and the Senate.
 
-2: The Privilege of the Writ of Habeas Corpus shall not be suspended, unless when in Cases of Rebellion or Invasion the public Safety may require it.
+2: No Bill of Attainder or ex post facto Law shall be passed.
 
-3: No Bill of Attainder or ex post facto Law shall be passed.
+3: No Capitation, or other direct, Tax shall be laid.
 
-4: No Capitation, or other direct, Tax shall be laid, unless in Proportion to the Census or Enumeration herein before directed to be taken.7
+4: No Tax or Duty shall be laid on Articles exported from any State, unless such Tax or Duty shall be imposed uniformly upon all States.
```

Comment:
The Congress has the authority to lay internal tariffs, but they cannot be put in place explicitly to overburden or underburden one state. Nonetheless these tariffs may have a harsher impact on states that are more economically entangled with their neighbors. There is no way to avoid that sort of flaw without exposing the ability to overburden one state as a form of economic punishment.